frontend/src/components/commons/ProgressPane.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import ProgressNode from "./ProgressNode";
+
+class ProgressPane extends Component {
+  static propTypes = {
+    progressSpecs: PropTypes.array.isRequired,
+    currentNode: PropTypes.number.isRequired,
+    paneTitle: PropTypes.string, //optional title parameter
+    exitButton: PropTypes.object // optional exit button, to leave the ProgressPane
+  };
+
+  state = {
+    currentNode: this.props.currentNode,
+    currentBody: this.props.progressSpecs[this.props.currentNode].body
+  };
+
+  changeNode(diff) {
+    const index = this.state.currentNode + diff;
+    this.setState({ currentNode: index, currentBody: this.props.progressSpecs[index].body });
+  }
+
+  render() {
+    return (
+      <div className="progress-pane emib-text-zone">
+        <div aria-label="progress" className="step-indicator">
+          <ul className="steps">
+            {this.props.progressSpecs.map(tab => (
+              <ProgressNode
+                key={tab.id}
+                id={tab.id}
+                text={tab.text}
+                current={this.state.currentNode}
+              />
+            ))}
+          </ul>
+        </div>
+        {this.props.paneTitle ? <h1 className="progress-pane">{this.props.paneTitle}</h1> : null}
+        {this.state.currentBody}
+        <div className="centeredButtons">
+          {this.state.currentNode < this.props.progressSpecs.length - 1 && (
+            <button type="button" className="btn btn-primary" onClick={() => this.changeNode(1)}>
+              Next to {this.props.progressSpecs[this.state.currentNode + 1].text}
```

Comment:
You should use the LOCALIZE variables to input this text to have the button names in English and in French.
Also, you'll have to figure out a way to put the currentNode in English and in French using the LOCALIZATION variables as well.

Reply:
As discussed offline, the translation of _Next to_ is kind of weird. So as agreed, replace that for _Next_.
Use the LOCALIZATION to get it in both languages.

Reply:
There is also a bug related to the display of these buttons (Next to and Back to).

**From my laptop screen:**

![image](https://user-images.githubusercontent.com/23021242/53195687-2d47b300-35e4-11e9-9457-eaa404631f64.png)

The buttons look good. Same as the wireframes.

**From my monitor:**

![image](https://user-images.githubusercontent.com/23021242/53195727-4a7c8180-35e4-11e9-9b2e-cc2f9cc60e8c.png)

The buttons are supposed to be on over the other.

Reply:
Fixed in 6d2e390; simplified button to read "Next"/"Suivant" and "Back"/"Retour" as the wording was a bit awkward in both languages.